### PR TITLE
Add .gitignore file

### PR DESCRIPTION
**Pull request by jart**
I noticed you were missing it.

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -19,3 +19,4 @@ Jake Voytko <jake@reviewninja.com>
 Kristina Chodorow <kchodorow@google.com>
 Lukacs Berki <lberki@google.com>
 Yuki Yugui Sonoda <yugui@yugui.jp>
+Justine Alexandra Roberts Tunney <jart@google.com>
```
Comment (pmbethe09) [inline CONTRIBUTORS]: Move up above Kristina -- this file is alphabetical on first name.

**Comment by pmbethe09** [comment]
LGTM after sorting CONTRIBUTORS.
I also want to know why tests failed for no clear reason, but that won't stop this request.

File: CONTRIBUTORS (hunk 1 above)
Comment (jart) [inline CONTRIBUTORS]: Done

**Comment by jart** [comment]
PTAL

Darwin tests on Jenkins have been failing for all bazelbuild projects lately. It should be unrelated to this PR.

**Pull request merged**